### DISCUSSION TITLE
Energia słoneczna użyta na własne potrzeby

### DISCUSSION THREAD

**Discussion by thewatcherpl** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#131] [Q&A]
Czy wyświetla Wam się Energia słoneczna użyta na własne potrzeby? Nie wiem może coś źle zrobiłem, od niedawna rozpoczynam przygodę z HA:) ale zawsze wskazuje mi "0 %" . Widać po słupkach, że w godzinach produkcji było zużycie na potrzeby własne, niestety nie widać już tego na skali procentowej 

<img width="1197" alt="energia" src="https://user-images.githubusercontent.com/130372533/235485917-81359736-73d0-48ab-a8be-b2c4511844cc.png">

**Comment by Jordan87** [comment]
U mnie jest ok, co dodales jako encje do zakladki energia jako siec elektryczna i panele sloneczne?
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/10414918/fb3f8d78-bc9f-44b1-9ff9-53c8dd1372f8)

![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/10414918/e3323f50-873a-469d-bd29-94acf2171bc5)

Podejrzewam ze jako panele sloneczne podales to samo co w produkcja do sieci i dlatego pokazuje ze wszystko co generujes to wysylasz i masz 0 autokonsumpcji.
